src/core/Akka.Remote/Transport/TestTransport.cs

```diff
@@ -590,7 +638,7 @@ public override bool Write(ByteString payload)
             if (Writeable)
             {
                 var result = _transport.Write(this, payload);
-                result.Wait();
+                result.Wait(TimeSpan.FromSeconds(3));
```

Comment:
Added a 3-second timeout to bring this inline with the JVM: https://github.com/akka/akka/blob/master/akka-remote/src/main/scala/akka/remote/transport/TestTransport.scala#L165